charts/rstudio-connect/NEWS.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 0.7.6
```

Comment:
Odd that 0.7.5 is missing from the NEWS.

Reply:
That's because I had to bump the chart version when I fixed the news: https://github.com/rstudio/helm/pull/536/files

I probably should have added an empty news header even though there were no changes to the chart